client/components/mma/shared/benefits/BenefitsConfiguration.ts

```diff
@@ -27,10 +31,37 @@ const guardianWeekly = {
 	description: 'Print magazine delivered to your door every week',
 };
 
+const partnerOffers: ProductBenefit = {
+	name: 'Exclusive access',
+	description: ' to partner offers',
+	specificToRegions: ['AUD'],
+};
+
 export interface ProductBenefit {
 	name?: string;
 	description?: string;
 	isUnavailable?: boolean;
+	specificToRegions?: CurrencyIso[];
+}
+
+export function filterBenefitByRegion(
+	benefit: {
+		specificToRegions?: CurrencyIso[];
+	},
+	currencyIso: string,
+) {
+	if (isCurrencyIso(currencyIso)) {
+		if (benefit.specificToRegions !== undefined) {
+			return benefit.specificToRegions.includes(currencyIso);
+		}
+
+		return true;
```

Comment:
Not sure if this is more succinct or if keeping the if statement is clearer 🤷 
```
return benefit.specificToRegions ? benefit.specificToRegions.includes(currencyIso) : true;
```

Reply:
had the same thought, erred on the side of clarity